packages/web3-http/package.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "web3-http",
```

Comment:
I think you should use existing http provider package in web3.js and modify/add new functionality in that instead of creating new package here.

Reply:
Hey are you suggesting modifying/adding functionality to the provider itself or adding a separate class to the provider package? Also I think that a provider as an abstraction is quite different to general http requests, and most of the other packages seem to have a single purpose. Happy to add the functionality to the provider package if you still think it should go there.